packages/xrpl/karma.config.js

```diff
@@ -0,0 +1,34 @@
+const webpackConfig = require('./test/webpack.config')[0]()
+delete webpackConfig.entry
+
+module.exports = function (config) {
+  config.set({
+    plugins: ['karma-webpack', 'karma-jasmine', 'karma-chrome-launcher'],
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+    webpack: webpackConfig,
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['jasmine'],
+
+    // list of files / patterns to load in the browser
+    files: [
+      'build/xrpl-latest.js',
+      'test/integration/index.ts',
+      'karma-setup.js',
+    ],
+
+    // preprocess matching files before serving them to the browser
+    // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
+    preprocessors: {
+      'karma-setup.js': ['webpack'],
+      // Use webpack to bundle our test files
+      'test/integration/index.ts': ['webpack'],
+    },
+
+    browsers: ['ChromeHeadless'],
```

Comment:
Just curious: I assume we could theoretically add additional browsers here? It'd be nice to (in the future, not in this PR) test on multiple browsers.

Reply:
Yes

https://www.npmjs.com/search?q=keywords:karma-launcher

http://karma-runner.github.io/6.4/config/browsers.html

Reply:
Is there a ticket for this? Seems like it'd be a good idea (and/or leave a TODO)

Reply:
Added on follow up list (to make tickets later)